src/exo/libs/gemm_acc_malloc.h

```diff
@@ -0,0 +1,11 @@
+#define HEAP_SIZE 100000
+#define DIM 16
```

Comment:
If these constants are not needed by calling code, should they instead be defined in the `.c` file?

Reply:
Note that there are ways to allow for these values to be overridden using compiler flags.  e.g. it may be better to write
```
#ifndef HEAP_SIZE
#define HEAP_SIZE 100000
#endif
```

Furthermore, you could run into trouble with name clashes given how generic these names are; e.g. `GEMMINI_ACC_HEAP_SIZE` would be a safer name?
(might be worth getting @alexreinking 's input on this)

Reply:
That's true, we never really need to change these parameters